serverlessworkflow/modules/ROOT/pages/testing-and-troubleshooting/mocking-opnapi-services-with-wiremock.adoc

```diff
@@ -223,9 +219,8 @@ class CountryServiceWorkflowTest {
 You can also start the WireMock server to be used in a specific test.
 
 .Prerequisites
-* A Serverless Workflow project is created. 
-+
-For more information about creating a workflow project, see link:{create-your-first-workflow-service}[Creating your first Serverless Workflow service].
+
```

Comment:
instead have 3 time the same statement, wouldn't be better to have only one at the beginning?

Reply:
[According to @hmanwani-rh](https://github.com/kiegroup/kogito-docs/pull/63#discussion_r921996025), each procedure should have prerequisites.